### DISCUSSION TITLE
Shared queue and public page to add songs to it (useful for parties)

### DISCUSSION THREAD

**Discussion by PilaScat** [music-assistant/support#2904] [Feature requests and ideas]
I would like to make a QR or NFC tag to allow my guests to add songs to the queue, maybe with a public page from which you can't manage the devices but only the queue
Spotify Jam type

**Comment by OzGav** [comment]
You could use HA to probably achieve this as there are so many options. Here is an example of one thing that can be done https://github.com/orgs/music-assistant/discussions/2175

**Reply by PilaScat** [reply]
I searched but found nothing like what I imagined

**Reply by OzGav** [reply]
Yeah you will need to create it from the tools you have. Here is another example of what can be done https://github.com/orgs/music-assistant/discussions/2601

**Reply by OzGav** [reply]
This appears to be a solution to this? https://github.com/DJS91/HAMusicAssistantJukebox

**Reply by PilaScat** [reply]
> This appears to be a solution to this? https://github.com/DJS91/HAMusicAssistantJukebox

Oh yeah, thanks for mentioning that
